cloud_storage_handler/api/elixircloud/csh/models.py

```diff
@@ -0,0 +1,44 @@
+"""Model for MinIO Configuration."""
+
+from typing import Annotated
+
+from pydantic import BaseModel, Field, constr
+
+
+class MinioConfig(BaseModel):
+    """Configuration for MinIO."""
```

Comment:
Please use Google-style docstrings ("Attributes" section) to describe model attributes (rather than `Field(description)`. You could do the same for "Examples" and get rid of `Field()` altogether for most params, making the model much more compact.